.github/scripts/userdata.sh

```diff
@@ -164,7 +164,7 @@ cat <<- EOF >> userdata_$1.sh
 sudo rm -rf plugins/opendistro_security
 sed -i /^opendistro_security/d config/elasticsearch.yml
 cd /opendistroforelasticsearch-kibana/
-sudo rm -rf plugins/opendistro_security
+sudo rm -rf plugins/opendistroSecurityKibana
 sed -i /^opendistro_security/d config/kibana.yml
```

Comment:
This has to be changed to sed -i /^opendistroSecurityKibana/d /etc/kibana/kibana.yml , as opendistro_security will no longer exist in the file.

Reply:
The settings have not changed. Just the plugin name